etc/sysctl.d/50-override.conf

```diff
@@ -55,6 +55,14 @@ net.ipv4.tcp_keepalive_time = 15
 net.ipv4.tcp_keepalive_intvl = 15
 net.ipv4.tcp_keepalive_probes = 4
 
+#
+# Increase the global network and TCP buffer sizes.
```

Comment:
This comment isn't very useful; it just re-states what the "code" already tells us. It'd be better to comment on _why_ we're setting these tunables to these values, since we can't determine that from the code.

Reply:
For this round it was just to be in parity with the illumos tunables so that's why I left vague. We will revisit this when we get to the performance epic.